### PR TITLE
feat: [#289] Remove github.com/ory/dockertest

### PR DESCRIPTION
**Pull request by hwbrzzl**
Closes https://github.com/goravel/goravel/issues/289 https://github.com/goravel/goravel/issues/261

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: database/console/main_test.go
Comment (hwbrzzl) [inline database/console/main_test.go:1]: The database docker only needs to be created once, the testing speed will be faster.

File: database/console/migrate_command_test.go [hunk 1]
```diff
@@ -20,19 +20,20 @@ type Agent struct {
...
 	}
 
 	var (
 		mockConfig *configmock.Config
-		pool       *dockertest.Pool
-		resource   *dockertest.Resource
 		query      ormcontract.Query
 	)
 
+	if err := testDatabaseDocker.Fresh(); err != nil {
```
Comment (hwbrzzl) [inline database/console/migrate_command_test.go:32]: Fresh the database, and clear data from other test cases.

**Review by devhaozi — APPROVED** [review]
LGTM

File: support/docker/redis.go [hunk 2]
```diff
@@ -2,41 +2,86 @@ package docker
 
 import (
 	"context"
+	"fmt"
+	"time"
 
 	"github.com/go-redis/redis/v8"
```
Comment (devhaozi) [inline support/docker/redis.go:8]: Q: Can we use `v9`?, v8 is out.

Comment (hwbrzzl) [reply support/docker/redis.go:8]: Good catch, will upgrade.

**Review by devhaozi — APPROVED** [review]
LGTM

**Pull request merged**